src/SeleniumLibrary/keywords/cookie.py

```diff
@@ -67,4 +81,22 @@ def add_cookie(self, name, value, path=None, domain=None, secure=None):
         # Secure should be True or False
         if is_truthy(secure):
             new_cookie['secure'] = secure
+        if is_truthy(expiry):
+            expiry_datetime = datetime.strptime(expiry, '%Y-%m-%d %H:%M:%S')
+            expiry_epoch = (expiry_datetime - datetime(1970, 1, 1)).total_seconds()
+            new_cookie['expiry'] = int(expiry_epoch)
         self.browser.add_cookie(new_cookie)
+
+
+class CookieInformation(object):
```

Comment:
First of all, such a generic `__init__` would be even better implemented like this:

```python
def __init__(self, data):
    self.__dict__.update(data)
```

I'm don't think that is a good idea in this context, though:
- Explicit is better than implicit. We are saving few lines of code but hiding information. Being able to use this object you needed to know what info the Selenium method returned.
- Making it sure we always return same info regardless what browsers return is important.

Reply:
Then something like this could work better:
```
class CookieInformation(object):

    def __init__(self, cookie):
        self.domain = cookie['domain'] if 'domain' in cookie else None
        self.expiry = cookie['expiry'] if 'expiry' in cookie else None
        self.httpOnly = cookie['httpOnly'] if 'httpOnly' in cookie else None
        self.name = cookie['name'] if 'name' in cookie else None
        self.path = cookie['path'] if 'path' in cookie else None
        self.secure = cookie['secure'] if 'secure' in cookie else None
        self.value = cookie['value'] if 'value' in cookie else None

    @property
    def full_info(self):
        return self.__dict__
```